### DISCUSSION TITLE
Link to user table

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5857] [Q&A]
I have created new module call: questions where i can make crud processes. I have developed it via basic laravel coding NOT Statamic blueprint. In create question i add title, text, upload file. How to add new field related to Statamic user model(table) and when i view the result in datatable it shows the user name and can search in datatable

**Comment by edalzell** [comment]
You've got your Statamic users in the database?

**Reply by HazemMohamed252** [reply]
Yes sure, but search not found it

**Reply by duncanmcclean** [reply]
How are you searching the users? Their name, their email?

**Reply by HazemMohamed252** [reply]
> How are you searching the users? Their name, their email?

Search for name
